### ISSUE TITLE
Open `Terms of Service` and `Copyright Policy` links in new tabs

### ISSUE THREAD

**Issue by VlodkoMr** [Joystream/atlas#1522]
Main menu disappeared from top navigation in 3 pages: 
Terms of Service: [http://joxi.ru/5mdLqDqUaG5akm](http://joxi.ru/5mdLqDqUaG5akm)
Copyright Policy: [http://joxi.ru/5mdLqDqUaG5akm](http://joxi.ru/5mdLqDqUaG5akm)
Studio: [http://joxi.ru/J2bWqwqfg9yq3A](http://joxi.ru/J2bWqwqfg9yq3A)

**Comment by kdembler** [comment]
I think we may need to split this into two separate issues (also they apply to the desktop version as well):
1. In general you can distinct two separate spaces in the app: viewer and studio. Both of those have separate topbars and side navigations. However, both the `Terms of Service` and `Copyright Policy` pages live somewhere in between viewer and studio as they're accessible from both of those. So the questions would be, which side navigation do we show for those pages? Viewer or studio? I don't think there's a good answer so we just don't show anything with only the logo allowing you to go to the homepage. I agree it's not ideal but at the same time I don't think it's worth introducing side navigation for those pages. I think one improvement we could do is open those pages in a separate tab - this way you're not interrupted in your current flow.
2. Regarding the lack of sidenav on the `Welcome to Studio` page, that's by design actually. Because, as mentioned above, Studio has its own separate navigation, we cannot really let user navigate to any page unless they go through the onboarding flow and sign in. That's why we don't show the sidenav there at all.

To sum up, I think the only actionable item here is to make `Terms of Service` page (and other similar) open in a new tab instead. I will update issue title to reflect that. If you disagree and think we should make other changes as well, feel free to reply and let me know what you think
